### DISCUSSION TITLE
Column sum [Sep 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3820] [Discord Threads]
> 🙋‍♂️ **georgew**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### `Column sum` not supported. You can use formula to get sum of cells across a row (horizontal).  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1019041955558215811)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
